migrations.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-docker-compose exec backend python manage.py makemigrations
-docker-compose exec backend python manage.py migrate
+docker-compose exec backend-wsgi python manage.py makemigrations
```

Comment:
shouldn't this be `backend` instead of `backend-wsgi`, since there's only 1 backend instance anymore?

Reply:
You are right, this is fixed now.